docker-compose/README.md

```diff
@@ -5,49 +5,47 @@ spinning up a network using docker-compose for local development.
 
 ## Usage
 
-Two parameterized `docker-compose` files are available:
+A parameterized Docker Compose file `bakers.yaml` is available:
```

Comment:
There are a number of environment variables that can be set, can you please refer to them in this readme (and a pointer to the node environment variables as well).

Reply:
done